packages/cubejs-snowflake-driver/driver/SnowflakeDriver.js

```diff
@@ -67,6 +69,13 @@ class SnowflakeDriver extends BaseDriver {
         WHERE columns.table_schema NOT IN ('INFORMATION_SCHEMA')
      `;
   }
+
+  async release() {
+    this.initialConnectPromise.then((connection) =>
```

Comment:
@TheSPD Promisification looks good to me! However, the promise itself is never returned from `release()`. You should either add `return` or rewrite this method using `await`.

Reply:
now returning the promise, thanks again. 
Not an expert in JS myself.